app/src/main/res/layout-land/hints_summary.xml

```diff
@@ -10,105 +10,81 @@
+      android:layout_height="48dp"
+      android:layout_gravity="center_vertical"
+      android:layout_marginStart="8dp"
+      android:layout_marginBottom="8dp"
+      android:contentDescription="@{@string/show_hide_hint_list(viewModel.hintsAndSolutionSummary)}"
+      android:padding="8dp"
+      android:src="@drawable/ic_arrow_drop_down_black_24dp"
+      android:visibility="@{viewModel.isHintRevealed? View.VISIBLE : View.GONE}"
+      app:isRotationAnimationClockwise="@{isListExpanded}"
+      app:layout_constraintBottom_toBottomOf="parent"
+      app:layout_constraintEnd_toEndOf="parent"
+      app:layout_constraintHorizontal_bias="0.82"
+      app:layout_constraintStart_toEndOf="@+id/hint_title"
+      app:layout_constraintTop_toTopOf="parent"
+      app:layout_constraintVertical_bias="0.0"
+      app:rotationAnimationAngle="@{180f}" />
 
-          <ImageView
-            android:id="@+id/hint_list_drop_down_icon"
-            android:layout_width="48dp"
-            android:layout_height="48dp"
-            android:padding="8dp"
-            android:layout_gravity="center_vertical"
-            android:contentDescription="@{@string/show_hide_hint_list(viewModel.hintsAndSolutionSummary)}"
-            android:src="@drawable/ic_arrow_drop_down_black_24dp"
-            app:isRotationAnimationClockwise="@{isListExpanded}"
-            app:rotationAnimationAngle="@{180f}" />
-        </FrameLayout>
-      </androidx.constraintlayout.widget.ConstraintLayout>
-
-      <LinearLayout
-        android:id="@+id/hint_list_container"
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:layout_marginStart="84dp"
-        android:layout_marginEnd="84dp"
-        android:layout_marginTop="8dp"
-        android:layout_marginBottom="32dp"
-        android:orientation="vertical"
-        android:visibility="@{isListExpanded? View.VISIBLE : View.GONE}">
+    <LinearLayout
+      android:id="@+id/hint_list_container"
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:layout_marginStart="84dp"
+      android:layout_marginTop="40dp"
+      android:layout_marginEnd="84dp"
+      android:layout_marginBottom="16dp"
+      android:orientation="vertical"
+      android:visibility="@{isListExpanded? View.VISIBLE : View.GONE}"
+      app:layout_constraintBottom_toBottomOf="parent"
+      app:layout_constraintTop_toBottomOf="@id/hint_title">
 
-        <TextView
-          android:id="@+id/hints_and_solution_summary"
-          android:layout_width="wrap_content"
-          android:layout_height="wrap_content"
-          android:layout_gravity="start"
-          android:fontFamily="sans-serif"
-          android:textColor="@color/oppiaPrimaryText"
-          android:textSize="16sp" />
-      </LinearLayout>
+      <TextView
+        android:id="@+id/hints_and_solution_summary"
+        style="@style/Body"
+        android:layout_gravity="start" />
```

Comment:
This view appears to be missing constraints.

Reply:
Actually this is wrapped in a LinearLayout and the constraints are there. But I think the LinearLayout isn't required, so I'm removing it and adding constraints to the TextView itself.

Reply:
Updated.